docs/user-guide/layer-settings.md

```diff
@@ -73,14 +73,29 @@ In particular, the user is allowed to:
 
 * Enable/disable the transparency for that layer
 
-* Enable/disable the use of the layer cached tiles  (if checked, the *Tiled=true* URL parameter will be added to the WMS request and to [use tiles cached with GeoWebCache](https://docs.geoserver.org/latest/en/user/geowebcache/using.html#direct-integration-with-geoserver-wms))
-
 * Decide to display the image as a single tile or as multiple tiles
 
 * Enable/disable the localized style. If enabled allows to include the MapStore's locale in each **GetMap**, **GetLegendGraphic** and **GetFeatureInfo** requests to the server, as explained in the [WMS Catalog Settings](catalog.md#wmswmts-catalog)
 
 * Enable/disable the *Force proxy* layer option. If enabled, forces the application to check the source and applies proxy if needed.
 
+* Enable/disable the use of the layer cached tiles. If checked, the *Tiled=true* URL parameter will be added to the WMS request and to [use tiles cached with GeoWebCache](https://docs.geoserver.org/latest/en/user/geowebcache/using.html#direct-integration-with-geoserver-wms).
+When the *Use cache options* is enabled, the **Available tile grids information** <img src="../img/button/update_button.jpg" class="ms-docbutton"/> and the **Remote custom tile grids** <img src="../img/button/tile_grid.jpg" class="ms-docbutton"/> buttons are also available to inform the user about the status of the grid set used in the map compared to those available remotely.
+
+!!!note
+
+    When the **Refresh Available Tile Grids** <img src="../img/button/update_button.jpg" class="ms-docbutton"/> button is checked, the info icon <img src="../img/button/info_ion.jpg" class="ms-docbutton"/> appears to inform the user about the default grid set parameters and whether they meet the remote grid sets.
+    
+    <img src="../img/layer-settings/default_gridset_info.jpg" class="ms-docimage"  style="max-width:300px;"/>
+    
+    When the **Remote custom tile grids** button is enabled, it turns green <img src="../img/button/tile_grid_green.jpg" class="ms-docbutton"/>, and the info icon <img src="../img/button/info_ion.jpg" class="ms-docbutton"/> appears to inform the user about the custom grid set parameters and whether they meet the remote grid sets.
+
+    <img src="../img/layer-settings/green_info.jpg" class="ms-docimage"  style="max-width:300px;"/>
+
+    The Info panel shows a warning message if it's not possible to find, on the remote side, a tile grid that matches the map projection, the selected tile size or the tile format. In this case, the user can change the [projection of the map](footer.md#crs-selector) or select different [tile size and tile format](layer-settings.md#display).
```

Comment:
We have to also include a warning in the same notes or just below saying what follow:

"The Gridset compatibility check made by MapStore which result is showed by the Info tooltip are usually quite reliable but should be anyway considered only to provide general matching indicators aimed to highlight possible compatibility issues between the current layer/map settings and the remote Tile Grid. Due to also the cache tolerance considered on the server side by GWC, it might even happen in some cases that the settings available on the client side don't HIT the tile cache even if all the checks listed are successful"